lib/src/boilerplate_suggestors/boilerplate_utilities.dart

```diff
@@ -119,3 +120,14 @@ void migrateClassToMixin(ClassDeclaration node, YieldPatch yieldPatch,
   propsAndStateClassNamesConvertedToNewBoilerplate[originalPublicClassName] =
       newMixinName;
 }
+
+/// Returns the name of the props class for a given [factoryDeclaration].
+String getPropsClassNameFromFactoryDeclaration(
+    TopLevelVariableDeclaration factoryDeclaration) {
+  return factoryDeclaration.variables.type.childEntities
+      .whereType<TypeArgumentList>()
+      .single
+      .arguments
+      .single
+      .toString();
```

Comment:
I believe `toSource()` is preferred over `.toString()`
```suggestion
      .toSource();
```